### ISSUE TITLE
Feature request: Theme for Visual Studio Code

### ISSUE THREAD

**Issue by tdoerrich** [Mrcuve0/Aritim-Dark#11]
I understand if this is outside the scope of these themes (I use it for everything I can so far. :D), but I wanted to ask if there is any chance of seeing a Visual Studio Code Aritim Dark Theme? It feels like I have been able to make most of my other software fit in to the style, except for Visual Studio Code. :) 

If I have time later, I might try and port the theme myself, however, could you add the colors to the github page? So it's easier than reading through the code to find the colors if I could port it later. :)

**Comment by Mrcuve0** [comment]
Hey, thank you for your kind words!

Actually, Aritim-Dark is inspired by Ayu Dark theme, which I think was initially born as a theme for VSCode. So no need to port anything! :D

I use it everyday and it's awesome, but it can be found in many variations. The best I found is packaged under the [Beautiful UI Theme Extension](https://marketplace.visualstudio.com/items?itemName=swashata.beautiful-ui).

Hope this helps, let me know! 

PS: If you have an account, you may want to leave a rating on my theme at the following page, it means a lot to me, thanks! --> https://www.pling.com/p/1281836/

**Comment by tdoerrich** [comment]
Oh. I missed that. Thanks for the link. Then I "just" need to make a theme for Joplin (digital notepad) and uLauncher. :)

**Comment by Mrcuve0** [comment]
Ping me once you complete them! I'd be glad to add a link section in the README file and point to your repository!

You can find all the colors in this file: 
> https://github.com/Mrcuve0/Aritim-Dark/blob/master/KDE/colorScheme/AritimDark.colors

You can also take a screenshot of a window and try to visually extract the colors directly from it:
> https://html-color-codes.info/colors-from-image/

**Comment by tdoerrich** [comment]
I'm currently working on the Joplin theme and just wanted to give an update (it's still not finished, but close).

Edited: Updated with correct text color

![image](https://user-images.githubusercontent.com/5912371/77518549-458ca000-6e7e-11ea-91f2-285875a5ddec.png)

**Comment by tdoerrich** [comment]
I have a question, I'm currently trying to make an image to better show the different colors used (and to find if something is missing like a warning color or such). However, I'm not sure what these colors are from the Konsole on one of your images:

![image](https://user-images.githubusercontent.com/5912371/77623075-bf389280-6f3f-11ea-91ef-cb479d5c8a2d.png)

Could you tell me if they are used somewhere and what for? E.G. Warnings or such?

And this is currently the image, if you have time, feel free to correct me or tell me if I have the wrong color somewhere. 

![AritimDark_Colors](https://user-images.githubusercontent.com/5912371/77623396-530a5e80-6f40-11ea-9471-2486267fb2d9.png)

The colors I haven't really found yet (that I believe is needed to make a full theme for things (like Joplin). :) are:

Error: Something dark red?
Warning: Something dark yellow?
Success: Something dark green?
Dark text color: For stuff that shouldn't have focus, I believe there is a color for this in the AYU theme I just haven't had time to look into that yet

Not sure if I'm taking too much of your time for this, then let me know and I will try and finalise it myself, but I wanted to ask since you are the original creator of the theme. :)

**Comment by Mrcuve0** [comment]
Don't worry, I really appreciate your commitment :D. 
Now, it's hard to recall what I did more than a year ago, but what I can tell you for sure is that the accent colors of the konsole theme do not follow a particular color scheme already existent (or maybe I modified the "Materia" konsole theme, I really don't remember). 

When I made it I simply tried to find nice colors with a good readability considering also blurred terminals, without considering consistency.

Yours is a really good idea tho: using the colors of the window decoration buttons seems to add additional consistency (which I reckon my konsole theme lacks for now ahah) to the entire theme.
The only problem may be the green, which isn't actually as dark as it should be (seems quite light to me).

BTW, feel free to define the colors yourself, I will adapt my konsole theme consequently.

In the following image, you can see the colors associated to "negative", "Neutral" and "Positive" text items. There's also the "Inactive Text" item, with a dark accent for text fonts.

![Screenshot_20200326_094554](https://user-images.githubusercontent.com/36898760/77627063-98ca2580-6f46-11ea-8235-ef6266e36e87.png)

**Comment by tdoerrich** [comment]
Ah! Thanks that is perfect! 

And I agree, the green is a bit too light (the red might be as well, need to test that). I'll keep working on this and post a new image once I'm finished with it! :)

**Comment by tdoerrich** [comment]
I've updated this with some new colors, however there are some I'm not too happy with (I'm def. not a great color person, so feel free to change or comment on anything).

Not too happy with or in need of: 
Warning, Orange: I don't feel like this fits with the rest of the colors, but I have been struggling with that one
The two text colors (without text), I feel like it needs a few more colors but I'm not too sure about those two. But needs colors for more markdown stuff like bold, links, etc.
Background colors: I love those two, but I feel like it needs to have a third with a tiny bit more different for when it is needed. 

![AritimDark_Colors](https://user-images.githubusercontent.com/5912371/77658017-60424000-6f76-11ea-84ee-0f119681427b.png)

This is the current Joplin theme I'm working on, and why I feel like it is "missing" some colors or such.
![image](https://user-images.githubusercontent.com/5912371/77658128-8831a380-6f76-11ea-8745-5d58137f658e.png)

Love to hear comments and suggestions if you have time.

**Comment by Mrcuve0** [comment]
You know, I find the orange quite pleasing if you ask me, I also find that it fits quite well with the rest of the colors. 
If this can help you or give you additional inspiration, the following piece of code has the original VScode Ayu theme enabled. (I'm not a fan of the violet keywords, but the orange is quite pleasing, don't you think?)

![Screenshot_20200327_105035](https://user-images.githubusercontent.com/36898760/77743732-ce3f4380-7018-11ea-8986-c4d820b9b181.png)

As for the additional background colors, you are right. Especially in your Joplin screenshot, a color that helps distinguish the "code" view and the "preview" view will be quite nice to have. Sadly I think this is a problem related to Joplin: the two sides probably share the same Qt element (is Joplin written in Qt by the way?) so they will always get the same color, I think.

In the following screenshot there's a detail of the Aritim-Dark Telegram theme I made some time ago (still to be completed 😅):
* The overall color is of course `#141A21`
* The Search bar background is `#242F3D`
* The saved Message background is `#17212B`

![Screenshot_20200327_110453](https://user-images.githubusercontent.com/36898760/77745185-2119fa80-701b-11ea-836d-fc9aea1686f2.png)

Maybe one of the last two could be your third color?

**Comment by tdoerrich** [comment]
Oh nice! Yeah, and I can change more colors in Joplin, it is an React (Electron) app. So it uses normal CSS for it to change. I needed to make some more changes to it. :) I will look into one of the ones from Telegram you have!

Also the orange you posted looks great! I'll add / use that one. What do you feel about the green and red?

**Comment by tdoerrich** [comment]
Updated with the other backgrounds, I think those adds some more nice alternatives!

I also kept the orange, because I felt like the AYU orange was a bit too light, however, I used it for the orange looking text.

![AritimDark_Colors](https://user-images.githubusercontent.com/5912371/77747882-ae5f4e00-701f-11ea-8e97-63ca31f600ab.png)

**Comment by Mrcuve0** [comment]
Hi! Another user wrote me a DM on Twitter saying that he implemented some application themes based on the color palette of Aritim-Dark. Hence, I planned to add an "Extras" section on the main README with all the links to all the "related" additional themes (like the ones you are preparing).

This is just to let you know that I'll be very happy to include also your themes in that section, keep me informed when you are done!

**Comment by tdoerrich** [comment]
Yep! I'm still waiting on some approvals, but I'll update tomorrow with some links to a ulauncher theme and such I have done. :)

**Comment by tdoerrich** [comment]
Hello again!

I have finished two things so far.

For [uLauncher](https://ulauncher.io/): https://github.com/TobiasDev/Ulauncher-Themes
For [Joplin ](https://github.com/laurent22/joplin)(custom CSS): https://github.com/TobiasDev/Joplin-Custom-CSS

I have also created a complete app them for Joplin, however, this hasn't been approved in the main app yet. Hopefully it will be soon! :)

**Comment by tdoerrich** [comment]
Also my pull-request was just accepted and merged into Joplin: https://github.com/laurent22/joplin/pull/2997 so as of the next release Joplin will include the "Aritim-Dark" theme. :)

I'm also quite curios what themes the other person that contacted you did! :) Might be more apps I use. :D

**Comment by Mrcuve0** [comment]
I'm updating the README right now! Really cool!

> so as of the next release Joplin will include the "Aritim-Dark" theme. :)

Sorry, as I don't use Joplin, what does that mean? Can it be selected as a theme directly from the program?

> I'm also quite curios what themes the other person that contacted you did! :) Might be more apps I use. :D

You'll see in a minute! 😁

**Comment by tdoerrich** [comment]
> Sorry, as I don't use Joplin, what does that mean? Can it be selected as a theme directly from the program?

Yeah, it will be part of the theme settings inside the app meaning you don't have to build it yourself! :) Also it is a fantastic multiplatform note-app with encryption! If you need one. :D 

![image](https://user-images.githubusercontent.com/5912371/78653681-c5c1f500-78c3-11ea-9cec-4738a898fdc3.png)

**Comment by Mrcuve0** [comment]
Here we are! Thank you once again, you can find it on the main README file.

![Screenshot_20200407_114831](https://user-images.githubusercontent.com/36898760/78655063-c2c80400-78c5-11ea-81f3-a83f484e56af.png)
